### DISCUSSION TITLE
DBeaver and Solr JDBC driver

### DISCUSSION THREAD

**Discussion by Jeetah** [dbeaver/dbeaver#572] [Q&A] [answered]
I tried to get DBeaver running against SolR with JDBC driver grabbed from there like described. However I have not been able to get any connection. I always get this error:

Unexpected driver error occurred while connecting to database
  java.util.concurrent.TimeoutException: Could not connect to ZooKeeper local-solr:8983/ within 10000 ms

Full connection string is: jdbc:solr://local-solr:8983/?collection=gta_de

Solr is running fine under this URL: http://local-solr:8983/solr/#/~cores/gta_de

Has anybody managed to get DBeaver running against Solr via JDBC?

**Comment by serge-rider** [comment, marked as answer]
Solr JDBC uses Zookeeper to operate with server.
Default Solr Zookeeper port is 9983 (8983 is a web interface port).

**Comment by Jeetah** [comment]
Thanks a lot. Oversaw the Zookeeper Ports.

**Comment by Pramodima** [comment]
But my exact connection string is:   jdbc:solr://xxxx:9983/?collection=client_sku
But  I am getting the below error message

Unexpected driver error occurred while connecting to database
  java.util.concurrent.TimeoutException: Could not connect to ZooKeeper xxx/ within 10000 ms
    Could not connect to ZooKeeper xxx:9983/ within 10000 ms
    Could not connect to ZooKeeper xxx:9983/ within 10000 ms

**Comment by peter-wang-wsl** [comment]
I use CDH6.3 and Solr7.4.0 and I config below url in dbeaver then connect successfully
`jdbc:solr://xxxx:2181/solr?collection=x`

**Comment by geekyouth** [comment]
![image](https://user-images.githubusercontent.com/12899262/80862572-b490bc00-8ca8-11ea-83c9-8633d31d5d38.png)
